packages/calcite-components/src/components/inline-editable/inline-editable.scss

```diff
@@ -5,6 +5,12 @@
  *
  * @prop --calcite-inline-editable-background-color: Specifies the background color of the component.
  * @prop --calcite-inline-editable-border-color: Specifies the border color of the component.
+ * @prop --calcite-inline-editable-button-corner-radius: defines the button's corner radius.
```

Comment:
This one isn't critical either as it can be addressed in https://github.com/Esri/calcite-design-system/pull/8888/, but I think we need 3 different sets of button props for this one:

1. [edit affordance](https://developers.arcgis.com/calcite-design-system/components/inline-editable/)
2. [confirm/cancel buttons](https://codepen.io/jcfranco/pen/bGJejzo?editors=100)

Each one of the above button instances is styled differently.